android/workmanager_faq.md

```diff
@@ -0,0 +1,11 @@
+# WorkManager FAQ
+The [WorkManager documentation](https://developer.android.com/reference/androidx/work/WorkManager) does not clearly explain all of the different edge cases that may be encountered. After some testing with this API, the edge cases that were tested are documented here for future reference:
+
+- When the device is turned off and turned on before the scheduled job, does the job complete?
```

Comment:
nit: "one time scheduled job", here & below

I'm concerned that one-time jobs & periodic jobs will be handled differently